scripts/common.py

```diff
@@ -217,6 +217,7 @@
     'logged-in-user-tests/click-all-buttons-on-about-page',
     'logged-in-user-tests/click-all-buttons-on-about-foundation-page',
     'logged-in-user-tests/click-all-buttons-on-thanks-for-donating-page',
+    'logged-in-user-tests/try-to-add-exploration-to-play-latee-give-feedback-and-report-it', # pylint: disable=line-too-long
```

Comment:
Avoid using bypass flag!
```suggestion
    'logged-in-user-tests/'
    'try-to-add-exploration-to-play-latee-give-feedback-and-report-it', # pylint: disable=line-too-long
```